diff -uNr BasiliskII-0.8.vanila/src/Unix/Makefile.in BasiliskII-0.8/src/Unix/Makefile.in
--- BasiliskII-0.8.vanila/src/Unix/Makefile.in	Tue Oct 26 18:56:38 1999
+++ BasiliskII-0.8/src/Unix/Makefile.in	Tue Aug 29 23:21:03 2000
@@ -63,13 +63,13 @@
 	cd Linux/NetDriver; make
 
 install: $(APP) installdirs
-	$(INSTALL_PROGRAM) $(APP) $(bindir)/$(APP)
-	-$(INSTALL_DATA) $(APP).1 $(man1dir)/$(APP).1
-	$(INSTALL_DATA) keycodes $(datadir)/$(APP)/keycodes
-	$(INSTALL_DATA) fbdevices $(datadir)/$(APP)/fbdevices
+	$(INSTALL_PROGRAM) $(APP) $(DESTDIR)$(bindir)/$(APP)
+	-$(INSTALL_DATA) $(APP).1 $(DESTDIR)$(man1dir)/$(APP).1
+	$(INSTALL_DATA) keycodes $(DESTDIR)$(datadir)/$(APP)/keycodes
+	$(INSTALL_DATA) fbdevices $(DESTDIR)$(datadir)/$(APP)/fbdevices
 
 installdirs:
-	$(SHELL) mkinstalldirs $(bindir) $(man1dir) $(datadir)/$(APP)
+	$(SHELL) mkinstalldirs $(DESTDIR)$(bindir) $(DESTDIR)$(man1dir) $(DESTDIR)$(datadir)/$(APP)
 
 uninstall:
 	rm -f $(bindir)/$(APP)
